.ocr/commands/show.md

```
@@ -0,0 +1,25 @@
+---
+description: Display a past OCR review session.
+name: "OCR: Show"
+category: Code Review
+tags: [ocr, history, sessions]
+---
+
+**Usage**
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)